tests/test_endpoints.py

```diff
@@ -45,6 +45,9 @@ def test_wait_for_ping(nhsd_apim_proxy_url):
     if resp.status_code == 429:
```

Comment:
We should import the Error_Handler here for these tests instead of having the same logic here